packages/nextjs/src/server/authMiddleware.ts

```diff
@@ -223,17 +236,25 @@ export { authMiddleware };
 const createDefaultAfterAuth = (
   isPublicRoute: ReturnType<typeof createRouteMatcher>,
   isApiRoute: ReturnType<typeof createApiRoutes>,
-  params: AuthMiddlewareParams,
+  options: { signInUrl: string; signUpUrl: string; publishableKey: string; secretKey: string },
```

Comment:
❓ why are we defining the options here explicitly? I think you're trying to minimize what is visible to this function, which is great, but in this case I think it's better to pass the options through so we don't have to worry about maintaining an extra inline type :)

Reply:
If we were to pass the params type, we would have to assert the necessary keys again to be safe, and that is just not the purpose of this function.